spring-session-samples/spring-session-sample-boot-hazelcast/spring-session-sample-boot-hazelcast.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'io.spring.convention.spring-sample-boot'
+
+dependencies {
+    compile project(':spring-session-hazelcast')
```

Comment:
There is some inconsistent indentation here.

Reply:
This is so strange – when I look at this in IntelliJ, I don't see the weird spacing. Not sure why my IDE isn't reflecting this. Thanks for catching it